translate/v23tov30/v23tov30.go

```diff
@@ -485,6 +487,53 @@ func translateFiles(files []old.File, m map[string]string) (ret []types.File) {
 	return
 }
 
+func translateNetworkd(units []old.Networkdunit, m map[string]string) []types.File {
+	var ret []types.File
+
+	for _, u := range units {
+		if u.Contents != "" {
+			file := types.File{
+				Node: types.Node{
+					// 2.x files are overwrite by default
+					Overwrite: util.BoolP(true),
+				},
+				FileEmbedded1: types.FileEmbedded1{
+					// Ignition default file permission
+					Mode: util.IntP(int(0644)),
+				},
+			}
+			// path /etc/systemd/network is hardcoded in Ignition2.x codebase.
+			// TODO: customize this path at compilation time.
+			file.Node.Path = path.Join(m["root"], filepath.Join("/etc/systemd/network", u.Name))
+
+			// URL encoding unit content to follow 'data' format - we could use base64 also.
+			file.FileEmbedded1.Contents.Source = util.StrPStrict("data:," + url.QueryEscape(u.Contents))
+
+			ret = append(ret, file)
+		}
+
+		for _, d := range u.Dropins {
+			file := types.File{
+				Node: types.Node{
+					// 2.x files are overwrite by default
+					Overwrite: util.BoolP(true),
+				},
+				FileEmbedded1: types.FileEmbedded1{
+					// Ignition default file permission
+					Mode: util.IntP(int(0644)),
+				},
+			}
+
+			file.Node.Path = path.Join(m["root"], filepath.Join("/etc/systemd/network", string(u.Name)+".d", d.Name))
```

Comment:
What is the reason for mixing `path.Join` and `filepath.Join`? Maybe it could be also:
```suggestion
			file.Node.Path = filepath.Join(m["root"], "/etc/systemd/network", string(u.Name)+".d", d.Name)
```

Reply:
aha - it seems I was not really awake this morning. Thanks for the heads-up.